products/iam/api.yaml

```diff
@@ -16,7 +16,10 @@
 --- !ruby/object:Api::Product
 name: Google Cloud IAM
 prefix: giam
-base_url: https://iam.googleapis.com/v1/
+versions:
+  - !ruby/object:Api::Product::Version
+    name: v1
+    base_url: https://iam.googleapis.com/v1/
```

Comment:
`default=true`? and for all the other files too. Unless you implement the logic I suggested for `default_version`